src/Resources/MeshId.h

```diff
@@ -0,0 +1,23 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "3D/AllMeshes.h"
+
+template <>
+struct fmt::formatter<openblack::MeshId>: fmt::formatter<uint32_t>
+{
+	auto format(openblack::MeshId c, format_context& ctx) { return formatter<uint32_t>::format(static_cast<uint32_t>(c), ctx); }
+};
+
+static entt::id_type MeshIdToResourceId(openblack::MeshId id)
```

Comment:
Warnings about this function

Reply:
![image](https://user-images.githubusercontent.com/32263167/163685647-7e5ade27-7da3-4ba1-929a-02a078e53610.png)

Reply:
So it's used but yeah this warning exists:
 Warning: /home/runner/work/openblack/openblack/src/Resources/MeshId.h:20:22: warning: unused function 'MeshIdToResourceId' [-Wunused-function]
  static entt::id_type MeshIdToResourceId(openblack::MeshId id)

Reply:
Try putting it in a namespace see if that helps